### PR TITLE
Feat/exception-simulation: support timeout simulation on the server side

### PR DESCRIPTION
**Pull request by Rhai2307**
Timeout ratio and timeout duration are taken from env 'TIMEOUT_RATIO'、'TIMEOUT_DURATION'

E.g. if you need 5% of requests to time out with at least 5s, set the env  to the following:

TIMEOUT_RATIO=0.05;
TIMEOUT_DURATION=5s

### CODE REVIEW

File: 3.0/adaptivesvc-triple/server/main.go [hunk 1]
```diff
@@ -23,12 +23,50 @@ import (
 	_ "dubbo.apache.org/dubbo-go/v3/imports"
 	"fmt"
 	"github.com/dubbogo/dubbo-go-benchmark/3.0/adaptivesvc-triple/api"
+	"math/rand"
+	"os"
+	"strconv"
```
Comment (justxuewei) [inline 3.0/adaptivesvc-triple/server/main.go:25]: Please format imports using "imports-formatter" which you may refer to [dubbogo/tools](https://github.com/dubbogo/tools).

**Review by justxuewei — COMMENTED** [review]
LGTM! Thanks for your contribution, there is a style problem, please fix that.

File: 3.0/adaptivesvc-triple/server/main.go (hunk 1 above)
Comment (Rhai2307) [reply 3.0/adaptivesvc-triple/server/main.go:25]: Fixed!

**Review by justxuewei — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by sheny1xuan — APPROVED** [review]

**Comment by Rhai2307** [comment]
apache/dubbo-go#1834

**Pull request merged**